packages/manager/cypress/support/intercepts/load-balancers.ts

```diff
@@ -103,6 +104,7 @@ export const mockUploadLoadBalancerCertificate = (
  *
  * @returns Cypress chainable.
  */
+// TODO: We should probably remove this mock and use "mockGetLoadBalancerServiceTargets" below.
```

Comment:
if this is an AGLB specific todo, could be worth switching to // TODO: AGLB to be more consistent with the other AGLB todo comments

```suggestion
// TODO: AGLB - We should probably remove this mock and use "mockGetLoadBalancerServiceTargets" below.
```